libraries/AP_HAL_ChibiOS/hwdef/3DR-L431/hwdef.inc

```diff
@@ -0,0 +1,87 @@
+# HW definition file for 3DR CAN Nodes (STM32L431)
+
+# MCU class and specific type
+MCU STM32L431 STM32L431xx
+
+# bootloader occupies 64k (48k + 16k, bootloader + STORAGE_FLASH)
+FLASH_RESERVE_START_KB 64
+FLASH_SIZE_KB 256
+
+# store parameters in pages 24 and 25
+STORAGE_FLASH_PAGE 24
+define HAL_STORAGE_SIZE 1800
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 15
+define CH_CFG_ST_RESOLUTION 16
+
+# board ID. See Tools/AP_Bootloader/board_types.txt
+APJ_BOARD_ID AP_HW_3DR_L431
+
+# use LSE clock source
+OSCILLATOR_HZ 32768
```

Comment:
Not sure this is a good idea.  FWIU CAN requires good timing, there's only one other board using LSE merged at the moment AFAICS.

Probably works.